apps/system/utils/fscmd.c

```diff
@@ -328,9 +328,6 @@ static int tash_cat(int argc, char **args)
 				fscmd_buffer[read_size] = '\0';
```

Comment:
this line also looks redundant as we no longer print using %s

Reply:
@gSahitya-samsung Could you check @abhishek-samsung 's comment?

Reply:
Done